config/openshift/kustomization.yaml

```diff
@@ -11,7 +11,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-namespace: tekton-operator
+namespace: openshift-operators
```

Comment:
can we change name of the deployment also
`tekton-operator` -> `openshift-operators` [here](https://github.com/tektoncd/operator/pull/188/files?file-filters%5B%5D=.md&file-filters%5B%5D=.yaml&file-filters%5B%5D=No+extension&hide-deleted-files=true#diff-96e3e6e0bffc7c64829ce3a843433c565121ef34ea453ab422fa3956c9ea9521R22)

Reply:
done